src/deployments/runtime/src/ou-validation-events/policy-changes.ts

```diff
@@ -155,6 +155,12 @@ export const handler = async (input: PolicyChangeEvent) => {
       acceleratorOus: config.getOrganizationalUnits(),
       acceleratorPrefix,
     });
+
+    await scps.attachOrDetachPoliciesToAccounts({
```

Comment:
Not required. 👎

Reply:
Ace - just pushing an update with the changes removed. Once it's passed testing should be good. Thanks for the review :)